src/components/donorNavbar.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Box,
+  Flex,
+  HStack,
+  Link,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  MenuDivider,
+  useDisclosure,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+
+export default function DonorNavBar() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <Box px={10}>
+        <Flex h={16} alignItems="center" justifyContent="space-between">
+          <IconButton
+            size="md"
+            icon={isOpen ? <CloseIcon /> : <HamburgerIcon />}
+            aria-label="Open Menu"
+            display={{ md: 'none' }}
+            onClick={isOpen ? onClose : onOpen}
+          />
+          <HStack as="nav" spacing={2} display={{ base: 'none', md: 'flex' }}>
+            <Flex alignItems="center">
+              <Menu>
+                <MenuButton
+                  borderWidth={3}
+                  borderColor="black"
+                  px={3}
+                  py={1}
+                  // height="40px"
+                  // width="150px"
+                  // as="button"}
+                  variant="link"
+                  cursor="pointer"
+                  minW={0}
+                >
+                  Select Chapter
+                </MenuButton>
+                <MenuList>
+                  <MenuItem>Georgia</MenuItem>
+                  <MenuDivider />
+                  <MenuItem>Texas</MenuItem>
+                  <MenuDivider />
+                  <MenuItem>Kansas</MenuItem>
+                </MenuList>
+              </Menu>
+              <HStack
+                as="nav"
+                spacing={2}
+                display={{ base: 'none', md: 'flex' }}
+              >
+                <Link
+                  borderWidth={3}
+                  borderColor="black"
+                  px={2}
+                  py={1}
+                  // rounded={'md'}
+                  _hover={{
+                    textDecoration: 'none',
+                    bg: useColorModeValue('gray.200', 'gray.700'),
+                  }}
+                  href="/recipients/login"
+                >
+                  Recipient Login
+                </Link>
+
+                <Link
```

Comment:
You need to use nextJS's `Link` component to perform routing properly. Here's an article showing how to indicate next/chakra-ui's Link components together: https://jools.dev/using-nextjs-link-with-chakra-ui-link